model/src/main/kotlin/config/Excludes.kt

```diff
@@ -57,8 +57,7 @@ data class Excludes(
     /**
      * Return the [ScopeExclude]s for the provided [scope].
      */
-    fun findScopeExcludes(scope: Scope): List<ScopeExclude> =
-        scopes.filter { it.matches(scope.name) }
+    fun findScopeExcludes(scope: Scope): List<ScopeExclude> = scopes.filter { it.matches(scope.name) }
```

Comment:
Nit: The functions (i.e. function bodies) were one-liners already before. I'd say this is just a formatting fix that removes unnecessary line breaks.

Reply:
Done.